### PR TITLE
Update cacheability with what we should include in Runtime package store

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Keeping packages csv up to date with the runtime package store in .All metapackage. This will go into rel/2.0.0-preview1 and dev

### CODE REVIEW

File: packages/packages.csv [hunk 1]
```diff
@@ -74,7 +74,7 @@ Microsoft.AspNetCore.Mvc.RazorPages,ship,include
 Microsoft.AspNetCore.Mvc.Razor.ViewCompilation,ship,include
 Microsoft.AspNetCore.Mvc.TagHelpers,ship,include
 Microsoft.AspNetCore.Mvc.ViewFeatures,ship,include
-Microsoft.AspNetCore.Mvc.WebApiCompatShim,ship,include
+Microsoft.AspNetCore.Mvc.WebApiCompatShim,ship,exclude
```
Comment (JunTaoLuo) [inline packages/packages.csv:77]: We don't expect any apps targeting .NET Core to use this package right @rynowak ?

File: packages/packages.csv [hunk 2]
```diff
@@ -131,7 +131,7 @@ Microsoft.EntityFrameworkCore.Sqlite.Core,ship,include
 Microsoft.EntityFrameworkCore.Sqlite.Design,ship,include
 Microsoft.EntityFrameworkCore.SqlServer,ship,include
 Microsoft.EntityFrameworkCore.SqlServer.Design,ship,include
-Microsoft.EntityFrameworkCore.Tools,ship,exclude
+Microsoft.EntityFrameworkCore.Tools,ship,include
```
Comment (JunTaoLuo) [inline packages/packages.csv:134]: This is not actually a tool and it's used as a package reference.

File: packages/packages.csv [hunk 3]
```diff
@@ -172,7 +172,7 @@ Microsoft.Extensions.Logging.Abstractions,ship,include
 Microsoft.Extensions.Logging.AzureAppServices,ship,include
 Microsoft.Extensions.Logging.Console,ship,include
 Microsoft.Extensions.Logging.Debug,ship,include
-Microsoft.Extensions.Logging.EventLog,ship,include
+Microsoft.Extensions.Logging.EventLog,ship,exclude
```
Comment (JunTaoLuo) [inline packages/packages.csv:175]: this is a .NET Framework only package

**Review by muratg — APPROVED** [review]
Approved

**Pull request merged**